src/nft/components/bag/BagRow.tsx

```diff
@@ -99,11 +101,11 @@ export const BagRow = ({ asset, usdPrice, removeAsset, showRemove, grayscale, is
   const showRemoveButton = Boolean(showRemove && cardHovered && !isMobile)
 
   const assetEthPrice = asset.updatedPriceInfo ? asset.updatedPriceInfo.ETHPrice : asset.priceInfo.ETHPrice
-  const assetEthPriceFormatted = formatWeiToDecimal(assetEthPrice)
-  const assetUSDPriceFormatted = ethNumberStandardFormatter(
-    usdPrice ? parseFloat(formatEther(assetEthPrice)) * usdPrice : usdPrice,
-    true
-  )
+  const assetEthPriceFormatted = formatNumberOrString({ input: formatEther(assetEthPrice), type: NumberType.NFTToken })
```

Comment:
could we add an overload to `formatNumberOrString` that takes in a BigNumber and runs `formatEther` on it? I've noticed you've had to use this pattern several times

Reply:
ty for reminding me to do that